test/e2e/sagemaker/__init__.py

```diff
@@ -23,3 +26,23 @@
 
 # PyTest marker for the current service
 service_marker = pytest.mark.service(arg=SERVICE_NAME)
+
+def make_reference(resource_prural, resource_name, spec_file, replacements, namespace="default"):
+
+    spec = load_resource_file(
+        SERVICE_NAME, spec_file, additional_replacements=replacements
+    )
+    logging.debug(spec)
```

Comment:
Remove debug statement? Or remove in the body of the fixtures where this is used. I see its duplicated

Reply:
good catch!